### PR TITLE
Add alpha_values and Qs to info files

### LINKED ISSUE #228: Ekobox info file does not update AlphaS_Qs and AlphaS_Vals from template
Looking at https://github.com/NNPDF/eko/blob/73e438ef3c406144e2879b3b4d5430a82b143718/src/ekobox/info_file.py#LL57C5-L57C63
it seems that we are not updating the alpha_Q and alpha_values from the INFO file template. Do you know why?
@AleCandido @felixhekhorn 

@giacomomagni this is the reason of the problem you faced with `evolven3fit_new`

### PR DESCRIPTION
**Pull request by andreab1997**
Closes #228

### CODE REVIEW

File: src/ekobox/info_file.py [hunk 1]
```diff
@@ -54,5 +57,27 @@ def build(
     template_info["MTop"] = theory_card.quark_masses.t.value
     template_info["AlphaS_MZ"] = theory_card.couplings.alphas.value
     template_info["AlphaS_OrderQCD"] = theory_card.order[0]
-    # NB: Maybe I need the actual operator to obtain AlphaS_Qs
+    evmod = couplings.couplings_mod_ev(operators_card.configs.evolution_method)
+    quark_masses = [(x.value) ** 2 for x in theory_card.quark_masses]
```
Comment (AleCandido) [inline src/ekobox/info_file.py:61]: There will be a property for this, at some point...

File: src/ekobox/info_file.py [hunk 2]
```diff
@@ -54,5 +57,27 @@ def build(
...
+            * np.pi
+            * sc.a_s(
+                muf2,
+            )
+            for muf2 in operators_card.mu2grid
+        ],
+        dtype=float,
+    )
+    template_info["AlphaS_Vals"] = alphas_values.tolist()
+    template_info["AlphaS_Qs"] = np.array(operators_card.mu2grid, dtype=float).tolist()
```
[the remark this answers is not in the document]
Comment (andreab1997) [reply src/ekobox/info_file.py:82]: I am not sure, let me try

File: src/ekobox/info_file.py [hunk 3]
```diff
@@ -79,5 +79,5 @@ def build(
         dtype=float,
     )
     template_info["AlphaS_Vals"] = alphas_values.tolist()
-    template_info["AlphaS_Qs"] = np.array(operators_card.mu2grid, dtype=float).tolist()
+    template_info["AlphaS_Qs"] = operators_card.mu2grid
```
Comment (AleCandido) [inline src/ekobox/info_file.py:82]:
```suggestion
    template_info["AlphaS_Qs"] = operators_card.mu2grid.tolist()
```
In the current and upcoming version of EKO this is an array, so at least the `.tolist()` was actually required.

Comment (andreab1997) [reply src/ekobox/info_file.py:82]: It seems that in the test (at least) it is a grid because it was failing before. I can show you in the next commit

Comment (AleCandido) [reply src/ekobox/info_file.py]: Ah, weird...

Comment (andreab1997) [reply src/ekobox/info_file.py]: So this is the reason of [03c9bcd](https://github.com/NNPDF/eko/pull/229/commits/03c9bcd2de603e8d08ba53c185a8fec7031059d3)

**Review by AleCandido — APPROVED** [review]

**Comment by andreab1997** [comment]
Are you ok with this @felixhekhorn ?

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**